Shared/Samples/Show popup/README.md

```diff
@@ -0,0 +1,34 @@
+# Show popup
+
+Show a predefined popup from a web map.
+
+![Show popup screenshot](show-popup.png)
+
+## Use case
+
+Many web maps contain predefined popups which are used to display the attributes associated with each feature layer in the map, such as hiking trails, land values, or unemployment rates. You can display text, attachments, images, charts, and web links. Rather than creating new popups to display information, you can easily access and display the predefined popups.
+
+## How to use the sample
+
+Tap on the features to prompt a popup that displays information about the feature.
+
+## How it works
+
+1. Create a `Map` using a URL.
+2. Create a `MapView` with the `Map`.
+3. Use the `GeoViewProxy.identifyLayers(_:screenPoint:tolerance:returnPopupsOnly:maximumResultsPerLayer:)` method to identify the top-most features.
+4. Create a `PopupView` with the result's `popups`.
```

Comment:
```suggestion
3. Use the `GeoViewProxy.identifyLayers(_:screenPoint:tolerance:returnPopupsOnly:maximumResultsPerLayer:)` method to identify the top-most feature.
4. Create a `PopupView` with the result's first popup.
```
In the past the `AGSPopupsViewController` supports displaying multiple popups, but that support is not added yet. @mhdostal Can you help add a link to the discussion for multiple popup? I saw it somewhere but cannot recall where. Thanks.

Reply:
There is currently no planned support for multiple popups.  It was decided that clients could easily implement that themselves if desired.  I can't find an online discussion about it; I just remember a verbal conversation that it wasn't needed.

Reply:
Thanks for the clarification!